Tests/js/extensions/assets/ez-draftserversidevalidation-tests.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-draftserversidevalidation-tests', function (Y) {
+    Y.namespace('eZ.Test.DraftServerSideValidation');
+
+    var Assert = Y.Assert;
+
+
+    Y.eZ.Test.DraftServerSideValidation.CallbackCalledTestCase = {
+        "Should call the server side error callback on error": function () {
+            var fields = [],
+                serverSideErrorCallbackCalled = false;
+
+            this.view.fire(this.action, {
+                formIsValid: true,
+                fields: fields,
+                serverSideErrorCallback: function(serverSideErrors) {
+                    serverSideErrors.forEach(function(serverSideError) {
+                        Assert.isInstanceOf(
```

Comment:
this is not enough, you should also make sure the correct "things" where parsed.